src/renderer/pages/project/script_editor/beat_editor.vue

```
@@ -14,8 +14,30 @@
+        @blur="justSaveAndPushToHistory"
+        :placeholder="
+          t('beat.speaker.placeholder', {
+            speaker: beat?.speaker || t('ui.common.speaker'),
+            language: t('languages.' + lang),
+          })
+        "
+        class="min-h-8 resize-y"
+      />
+      <Button variant="outline" size="sm" @click="generateAudio()" class="w-fit" :disabled="beat?.text.length === 0">{{
+        t("ui.actions.generateAudio")
+        }}</Button>
```

ESLint

[error] Delete `··`

(prettier/prettier)